file_snapshot.go

```diff
@@ -240,7 +240,11 @@ func (f *FileSnapshotStore) getSnapshots() ([]*fileSnapshotMeta, error) {
 		// Ignore any temporary snapshots
 		dirName := snap.Name()
 		if strings.HasSuffix(dirName, tmpSuffix) {
-			f.logger.Printf("[WARN] snapshot: Found temporary snapshot: %v", dirName)
+			f.logger.Printf("[WARN] snapshot: Found temporary snapshot: %v, deleting..", dirName)
```

Comment:
You could add cleanup of temp snapshots after here, since this is the thread where snapshots get created, that way they will get periodically cleaned up:

https://github.com/hashicorp/raft/blob/library-v2-stage-one/snapshot.go#L72